src/sentry/auth/access.py

```diff
@@ -305,15 +310,15 @@ def from_request(request, organization=None, scopes=None):
             role=role,
         )
 
-    # TODO: from_auth does not take scopes as a parameter so this fails for anon user
```

Comment:
is this path unused then completely, or handled elsewhere? i worry when i see a sensitive change like this and not corresponding test changes/removals

Reply:
@dcramer any test on this code path would blow up because `scopes` isn't an argument. When I was adding tests for the auth code a while back, I had to skip this one because it's broken.

Reply:
@dcramer  It looks like this PR introduced the bug: https://github.com/getsentry/sentry/commit/3ca7780db0fc0d3085cb43192bb003e29da42d34#diff-c2095505f74f61d45f11177177072f38L351  

So maybe this used to work before that but I suspect this code path just isn't used anymore.